packages/engine-cli/src/engine-build.ts

```diff
@@ -176,17 +177,33 @@ export async function runEngine({
             console.log(`Runtime options: ${JSON.stringify(Object.fromEntries(runtimeOptions.entries()), null, 2)}`);
         }
 
-        runManagerContext = runNodeManager({
```

Comment:
runNodeManager doesn't seems to be used anymore

Reply:
removed